docs/content/getting-started/docker.md

```diff
@@ -48,3 +48,17 @@ For example, for running cardano-node + ogmios on the testnet, listening to tcp/
 ```console
 $ NETWORK=testnet OGMIOS_PORT=1338 docker-compose --project-name cardano-ogmios-testnet up
 ```
+
+## As a single image
+
+If you're not fond of docker-compose and the container orchestration it requires, you may also build (or pull) an image to run both a `cardano-node` and `ogmios` side-by-side, within the same container. Note that this is the default build target when building ogmios with Docker, but we can be explicit when building the image using `--target cardano-node-ogmios`. For example:
+
+```console
+$ docker build --target cardano-node-ogmios --tag ktorz/cardano-node-ogmios:latest . 
+```
+
```

Comment:
_Note that this is the default build target for the image, but `--target cardano-node-ogmios` can be set explicitly_